paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -125,7 +125,6 @@ internal abstract class BaseSheetViewModel(
             // Drop the first item, since we don't need to clear errors/mandates when there aren't any.
             navigationHandler.currentScreen.drop(1).collect {
                 clearErrorMessages()
-                mandateHandler.updateMandateText(mandateText = null, showAbove = false)
```

Comment:
This is the heart of the fix.

Reply:
Also, I find it weird that removing this line caused no tests to fail 🙈

Reply:
🙈 yikes